v4/export/sql.go

```diff
@@ -120,30 +124,51 @@ func SelectAllFromTable(conf *Config, db *sql.DB, database, table string) (Table
 	}, nil
 }
 
-func buildSelectAllQuery(conf *Config, db *sql.DB, database, table string) (string, error) {
+func buildSelectAllQuery(conf *Config, db *sql.DB, database, table string, where string, orderByClause string) (string, error) {
 	var query strings.Builder
+	var err error
 	query.WriteString("SELECT * FROM ")
 	query.WriteString(database)
 	query.WriteString(".")
 	query.WriteString(table)
+
+	separator := "WHERE"
+	if conf.Where != "" {
+		query.WriteString(" ")
+		query.WriteString(separator)
+		query.WriteString(" ")
+		query.WriteString(conf.Where)
+		separator = "AND"
+	}
+
+	if where != "" {
+		query.WriteString(" ")
+		query.WriteString(separator)
+		query.WriteString(" ")
+		query.WriteString(where)
+	}
```

Comment:
What about using a `[]string`, and join them with " AND "?

Reply:
`[]string` join is not seen in this repo. I think we need keep consistent is better

Reply:
I think we can make it simpler